### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - David Berger

### DISCUSSION THREAD

**Discussion by Davtek11** [heig-vd-dai-course/heig-vd-dai-course#322] [Show and tell]
https://github.com/Davtek11/DAI-Installation

**Comment by ludelafo** [comment]
Everything seems fine for me except the `pom.xml`. It seems you have removed the initial properties from the following lines:

```xml
<properties>
    <!-- Omitted for brevity -->
</properties>
```

You must keep the initial values to ensure Maven can encode the files correctly (more on this in the next course) and check for the right Java version.

Also, these lines can be removed as the `maven-shade-plugin` is used as a plugin (already set in the `plugins` tags) and not a dependency:

```xml
        <dependency>
            <groupId>org.apache.maven.plugins</groupId>
            <artifactId>maven-shade-plugin</artifactId>
            <version>3.6.0</version>
        </dependency>
```
